### ISSUE TITLE
Uploaded File in the email is 0byte, but uploaded correctly

### ISSUE THREAD

**Issue by Largo** [bolt/boltforms#134]
Branch: Master
Bolt: 3.2.4

Uploaded File in the email is 0byte, but it was uploaded correctly to the specified folder.
In Stable, the file is attached correctly, and the email is sent, but a error shows up (Can't find file im php tmp)

**Comment by GawainLynch** [comment]
> Can't find file im php tmp

Yeah, the file is moved prior to the email processor, so that is expected that you wouldn't find it in temp at that stage.

In 4.0 we've made the attachment optional:

```
my_form_name:
    notification:
        attach_files: true
```

**Comment by GawainLynch** [comment]
Added a note in the docs 75337070869e45d0e9315c82a7536b5f9f7af13d

**Comment by Largo** [comment]
What I meant was that the attached file in the email was 0bytes long, so it
was incorrectly attached to the email.
The upload went through correctly tough.

Freundliche Grüsse

Andreas Wildi

2017-01-21 13:46 GMT+09:00 Gawain Lynch <notifications@github.com>:

**Comment by GawainLynch** [comment]
Ah, yes and you are very correct. I was just running a Python SMTP debug server in my console and saw:
```
--_=_swift_v4_1484973701_1b50cfd3cada2516c665f22c62c5c9cc_=_
Content-Type: application/octet-stream; name=1.png.f04bee8ae09a930193991f66
Content-Transfer-Encoding: base64
Content-Disposition: attachment; filename=1.png.f04bee8ae09a930193991f66
```

… and assumed that was working … shall fix now!

**Comment by GawainLynch** [comment]
OK, set up a real SMTP server and properly fixed it in e747822b86704197e9626bc25085385123960bea  :+1:

**Comment by GawainLynch** [comment]
Oh, and that is included in 4.0.0-beta1
